internal/rego/oci/oci.go

```diff
@@ -177,8 +177,6 @@ func registerOCIImageFiles() {
 	rego.RegisterBuiltin2(&decl, ociImageFiles)
 }
 
-const maxBytes = 10 * 1024 * 1024 // 10 MB
```

Comment:
or just bump this to 30 MB?

Reply:
That seem arbitrary as well.

I wonder if making this configurable, e.g. via env var, is a better approach.

Reply:
What was the thinking behind adding a limit initially? Maybe there's still some value in it, even if it's bumped to 100MB or something.

Reply:
Let's try out without any restrictions. We can always go back later on if we find a good reason.